pkg/config/apiserver.go

```diff
@@ -219,15 +220,16 @@ func (a *API) register(c *restful.Container) {
 
 // NewAPI creates a new API server
 func NewAPI(version string, port uint16, tc expr.TypeChecker, aspectFinder AspectValidatorFinder,
-	builderFinder BuilderValidatorFinder, findAspects AdapterToAspectMapper, store store.KeyValueStore) *API {
+	builderFinder BuilderValidatorFinder, getBuilderInfoFns []adapter.GetBuilderInfoFn, findAspects AdapterToAspectMapper, store store.KeyValueStore) *API {
```

Comment:
Ug. Don't love this appearing here either (I know we just bounced it from `adaptermanager/manager` to here). I think as part of the validation refactoring I want to do we can change the interface so that the validator is wired up in `cmd/server/server.go` and passed in here, to avoid all this stuff. But this PR is fine as-is.